### ISSUE TITLE
Viewing order book, Only Remove offer available.  Enable and Edit would be convenient 

### ISSUE THREAD

**Issue by opticbit** [bisq-network/bisq#3903]
### Description

Viewing the Buy/Sell order book, looking at your own ad only has the option to remove.
including the option to enable, and edit would be helpful.

#### Version

1.2.5
### Steps to reproduce

view order book page

### Expected behaviour

button to edit and turn on/off the offer.

only button available is to remove the offer.

#### Device or machine

Mac OS 10.15.2
Ubuntu 18.4, 19.10

#### Additional info

plenty of space to split the button into 1/3s

**Comment by beingindot** [comment]
have you looked into portfolio / My open offers tab? there you can find edit button.
I think editing price is available which wont affect the security deposit etc.

**Comment by opticbit** [comment]
Yes.  I have 2 pages of offers.  I go back and forth looking at the order books, then my open offers, then edit the offer. 

Running bisq on VPS, so looking through vnc.

**Comment by opticbit** [comment]
> have you looked into portfolio / My open offers tab? there you can find edit button.
> I think editing price is available which wont affect the security deposit etc.

Yes. I have 2 pages of offers. I go back and forth looking at the order books, then my open offers, then edit the offer.

Running bisq on VPS, so looking through vnc.
